### DISCUSSION TITLE
Started working on implementing themes into quick reply

### DISCUSSION THREAD

**Discussion by Steve0Greatness** [theforumhelpers/theforumhelpers.github.io#275] [Show and tell]
as of currently, you need to switch them manually, but I it them working, you can view it on [replit](https://replit.com/@stevesgreatness/QuickReply)
also, can you give me ideas to use for color schemes(dark, scratch, and terminal).
@leahcimto, @Accio1, @BarelySmooth

update, as of currently, all themes have placeholder looks.

**Comment by aidan-crockett** [comment]
fix your link
it should be https://replit.com/@stevesgreatness/QuickReply

**Reply by Steve0Greatness** [reply]
done.

**Comment by Steve0Greatness** [comment]
@leahcimto, do you have any opinions on how I did one the Scratch theme seen on one of the links?

**Comment by Steve0Greatness** [comment]
@Accio1, do you think there's any better way to do this?
